TeslaSUI/TeslaSUI/Views/BottomSheetView.swift

```diff
@@ -0,0 +1,74 @@
+// BottomSheetView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Нижний список
+struct BottomSheetView<Content: View>: View {
+    let content: () -> Content
+    
+    @GestureState private var gestureOffset = CGSize.zero
+    @State private var currentMenuOffsetY: CGFloat = 0.0
+    @State private var lastMenuOffsetY: CGFloat = 0.0
+    
+    var dragGesture: some Gesture {
```

Comment:
```suggestion
    var dragGesture: some Gesture {
```

это используется только внутри, можно наверное тоже сделать приватной